### ISSUE TITLE
Refresh select menu

### ISSUE THREAD

**Issue by ckosmowski** [fnagel/jquery-ui#311]
I have a jquery-ui-selectmenu (jquery-ui-1.11.0pre) with only one option in it (for now). Now when i change the text of that value in the original select element i'll have to refresh the selectmenu element. So i do:

$jq("#selectMenu").first().text("newText");
$jq("#selectMenu").selectmenu("refresh");

It seems that now the text of the new option actually changed (in the dropdown menu) but the select value that i see on the button is not updated it is still the old value. How can i fix this?

**Comment by fnagel** [comment]
Hey, would you please provide a demo jsFiddle (see #61)?

This should work (if not I will fix this) but your issue could be related to using jQuery UI 1.11.x which is not supported by this branch. Please use the upcoming official Selectmenu instead, see: #140

**Comment by ckosmowski** [comment]
I'm having trouble getting the library i'm using into jsfiddle since i 
built it myself and i don't have a place available to upload at the 
moment. But i'm a bit confused about which branch is the right one. I 
downloaded the latest jquery-ui branch from github a few days ago and 
built it myself and that's which version i have.

Am 18.07.2013 09:58, schrieb Felix Nagel:

**Comment by fnagel** [comment]
That sounds like you're download jQueryUI from the official GitHub account,
right? That way you will have latest dev (master) version which will not
work with this fork.
If so, you should really try the official version:
https://github.com/jquery/jquery-ui/tree/selectmenu

tl;dr:
This fork's Selectmenu is for old jQuery UI versions (< 1.10.x)
The above linked version is for later versions (1.11.0-now)

**Comment by fnagel** [comment]
Any feedback on this issue?

**Comment by ckosmowski** [comment]
It's fixed with https://github.com/jquery/jquery-ui/commit/67f04e4e42e67c1b73a97efc1c46cf986cec4794 thanks a lot.

**Comment by fnagel** [comment]
Thanks for the feedback!
